src/diagnostics.md

```diff
@@ -449,8 +449,8 @@ The possible values of [`Applicability`][appl] are:
 
 - `MachineApplicable`: Can be applied mechanically.
 - `HasPlaceholders`: Cannot be applied mechanically because it has placeholder
-  text in the suggestions. For example, "Try adding a type: \`let x:
-  \<type\>\`".
+  text in the suggestions. For example: ``try adding a type: `let x:
+  <type>` ``.
```

Comment:
Seems this adds trailing whitespace, is there any way to avoid it?

Reply:
Not that I'm aware of; I think it's a limitation of Markdown. If someone has a suggestion for how to get rid of it, I'm happy to try it!